tcl/envmngt.tcl.in

```diff
@@ -1236,6 +1282,29 @@ ml <- function(...){
 }}
          }
       }
+      pwsh {
+         if {[getState is_win]} {
+            reportErrorAndExit {
+               No autoinit mode available for 'pwsh' shell on Windows}
```

Comment:
Please rewrite the following way to avoid trailing space in error message string:

```tcl
            reportErrorAndExit {No autoinit mode available for 'pwsh' shell\   
               on Windows}
```